tests/requirements-linting.txt

```diff
@@ -3,7 +3,7 @@ flake8==3.9.2
 flake8-quotes==3.2.0
 hypothesis==6.13.10
 isort==5.8.0
-mypy==0.812
+mypy==0.901
```

Comment:
Why not [`0.902`](https://pypi.org/project/mypy/0.902/)?

Reply:
0.901 was the newest when I started this, 0.902 only released Thursday past. It shouldn't cause problems to bump this.

Reply:
bumped version in 97efeee, no issues in the tests.